backend/fms_core/resources/transfer.py

```diff
@@ -0,0 +1,187 @@
+        skip_rows(dataset, 7)  # Skip preamble
+
+    def import_obj(self, obj, data, dry_run):
+        self.extracted_from = Sample.objects.get(
+            container__barcode=get_normalized_str(data, "Source Container Barcode"),
+            coordinates=get_normalized_str(data, "Source Location Coord"),
+        )
+        # Cast the "Source Depleted" cell to a Python Boolean value and
+        # update the original sample if needed. This is the act of the
+        # extracted sample depleting the original in the process of its
+        # creation.
+        self.extracted_from.depleted = (self.extracted_from.depleted or
+                                        check_truth_like(get_normalized_str(data, "Source Depleted")))
+
+        # Create a process for the current extraction
+        self.process = Process.objects.create(protocol=Protocol.objects.get(name="Transfer"),
+                                              comment="Sample Transfer (imported from template)")
+        
+        super().import_obj(obj, data, dry_run)
+
+    def import_field(self, field, obj, data, is_m2m=False):
+        if field.attribute in ('source_depleted', 'child_of'):
+            # Computed field, skip importing it.
+            return
+
+        if field.attribute == 'destination_container':
+            parent = None;
+            destination_parent_container_barcode = get_normalized_str(data, "Destination Parent Container Barcode")
+
+            if destination_parent_container_barcode:
+                shared_parent_info = dict(
+                    barcode=destination_parent_container_barcode,
+                )
+                try:
+                    parent = Container.objects.get(**shared_parent_info)
+                except Container.DoesNotExist:
+                    raise Exception("Destination parent container does not exist")
+
+            try:
+                obj.container = Container.objects.get(barcode=get_normalized_str(data, "Destination Container Barcode"))
+            except Container.DoesNotExist:
+                obj.container = Container.objects.create(
+                    barcode=get_normalized_str(data, "Destination Container Barcode"),
+                    location=parent,
+                    name=get_normalized_str(data, "Destination Container Name"),
+                    kind=get_normalized_str(data, "Destination Container Kind"),
+                    coordinates=get_normalized_str(data, "Destination Parent Container Coord"),
+                    comment=f"Automatically generated via transfer template import on "
+                            f"{datetime.utcnow().isoformat()}Z"
+                )
+
+            obj.coordinates = get_normalized_str(data, "Destination Location Coord")
+
+            return
+
+        if field.column_name == "Volume Used (uL)":
+            vu = blank_str_to_none(data.get("Volume Used (uL)"))  # "" -> None for CSVs
+            if vu:
+                self.volume_used = float_to_decimal(vu)
```

Comment:
you do not need to put this in self.volume_used if you put it in volume... the self.volume_used variable was to get the value to the process_sample, but since volume is part of the instance you can use that value instead to create your process_sample.

Reply:
Volume_used is still an attribute on the parent sample... I think it is right to modify it while we still keep it. But maybe I should add the volume currently used to the volume_used already stored...

Reply:
volume_used is to be removed ... I left it there until the extraction was done ... i must have forgotten to remove it in the wake of the repo migration... The volume_used was solely used with extractions so since it is move to ProcessSample the volume_used on sample should be removed ...